cypress/fixtures/analyticalObject.json

```diff
@@ -0,0 +1,117 @@
+{
+    "currentAnalyticalObject": {
+        "href": "http://localhost:8080/api/38/visualizations/UlfTKWZWV4u",
```

Comment:
Does it matter that the href includes localhost:8080 (for running on CI)?

Reply:
It shouldn't since this part is not used in Maps, I think the only part that is interesting for Maps is `columns`, `rows`.